databases/week3/homework3.txt

```diff
@@ -0,0 +1,175 @@
+Use mealsharing;
+
+CREATE TABLE `meal` (
+  `id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY
```

Comment:
okey, Flemming,.i will do that now,.,.tak

Reply:
Hi Charles. I've tried the CREATE TABLE statements again and there is still the same error. The error actually has nothing to do with the VARCHAR columns, even though it can look that way. It's perfectly fine to use VARCHAR(255) columns. The error is actually a syntax error because of a few missing things.

This is what's missing from the first line:
```suggestion
  `id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
```
There are other errors as well. See if you can fix them all by using the technique i described earlier.

Reply:
Tak, Flemming,.,let me review them now,,.,